src/redux/books/booksSlice.js

```diff
@@ -1,23 +1,54 @@
 import { createSlice } from '@reduxjs/toolkit';
 
+let nextItemId = 1;
+
+function getNextItemId() {
+  const newItemId = `item${nextItemId}`;
+  nextItemId += 1;
+  return newItemId;
+}
+
 const initialState = {
-  books: [],
+  books: [
+    {
+      item_id: 'getNextItemId()',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'getNextItemId()',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'getNextItemId()',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
 };
```

Comment:
[ OPTIONAL BUT HIGHLY RECOMMENDED ] 👋 Hi there! It looks like there's a small issue with the `item_id` values. You should call the `getNextItemId()` function to generate unique IDs for each book. Here's the corrected code:
```sh
const initialState = {
  books: [
    {
      item_id: getNextItemId(), 🆕 // Call the function to get the unique ID
      title: 'The Great Gatsby',
      author: 'John Smith',
      category: 'Fiction',
    },
    {
      item_id: getNextItemId(), 🆕 // Call the function again for a different ID
      title: 'Anna Karenina',
      author: 'Leo Tolstoy',
      category: 'Fiction',
    },
    {
      item_id: getNextItemId(), 🆕 // Call the function again for another different ID
      title: 'The Selfish Gene',
      author: 'Richard Dawkins',
      category: 'Nonfiction',
    },
  ],
};
```

- This will ensure that each book has a unique identifier. 📚

Reply:
Thanks for the feedback! I'll make the required changes and use getNextItemId() for unique IDs.